examples/llamaindex/Evaluate_RAG_with_LlamaIndex.ipynb

```diff
@@ -0,0 +1,550 @@
+{
```

Comment:
<p>High level comment: Cool notebook! I like the eval part especially. It'd be awesome to deep dive on that (as another task, another day)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>